presentation/src/main/java/com/yapp/bol/presentation/view/match/MatchViewModel.kt

```diff
@@ -2,17 +2,91 @@ package com.yapp.bol.presentation.view.match
 
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.yapp.bol.domain.usecase.login.MatchUseCase
+import com.yapp.bol.presentation.model.MemberItem
 import com.yapp.bol.presentation.utils.Constant.GAME_RESULT_RECORD
+import com.yapp.bol.presentation.utils.checkedApiResult
 import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.collectLatest
+import kotlinx.coroutines.launch
 import javax.inject.Inject
 
 @HiltViewModel
-class MatchViewModel @Inject constructor() : ViewModel() {
+class MatchViewModel @Inject constructor(
+    private val matchUseCase: MatchUseCase
+) : ViewModel() {
 
     private val _toolBarTitle = MutableLiveData(GAME_RESULT_RECORD)
     val toolBarTitle = _toolBarTitle
 
+    private val _members = MutableLiveData<List<MemberItem>>(listOf())
+    val members = _members
```

Comment:
```suggestion
    private val _members = MutableLiveData<List<MemberItem>>(listOf())
    val members = _members.asLiveData()
```
외부에 보여질 때 immutable 하게 만들기 위해 asLiveData() 로 감싸야해요!
참고 [link](https://velog.io/@changhee09/%EC%95%88%EB%93%9C%EB%A1%9C%EC%9D%B4%EB%93%9C-LiveData#:~:text=%EC%A0%9C%EA%B1%B0%ED%95%A0%20%EC%88%98%20%EC%9E%88%EC%8A%B5%EB%8B%88%EB%8B%A4.-,LiveData%20%EB%A7%8C%EB%93%A4%EA%B8%B0,-LiveData%EB%8A%94%20%EC%9C%84%EC%97%90%EC%84%9C)